CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## PyNWB 3.0.0 (Upcoming)
 
+### Deprecations
+- The following deprecated classes will now raise errors when creating new instances of these classes: ClusteringWaveforms, Clustering, SweepTable. Reading files using these data types will continue to be supported.
+- The following methods and arguments have been deprecated: ProcessingModule.add_container, ProcessingModule.get_container, ProcessingModule.add_data_interface, ProcessingModule.get_data_interface, ScratchData.notes, NWBFile.ic_electrodes, NWBFile.ec_electrodes, NWBFile.icephys_filtering, NWBFile.modules, ImageSeries.bits_per_pixel, ImageSeries.format, ImagingPlane.manifold, ImagingPlane.conversion, IndexSeries.unit, IndexSeries.indexed_timeseries
```

Comment:
If there are replacements for these deprecations, then it would be nice to either mention them briefly here or point to where the appropriate replacement can be found. 

Also, please add `` quotes for the functions, so that the names render as code.